azure/lb.sh

```diff
@@ -82,7 +82,7 @@ fn azure_lb_frontend_ip_set_subnet_vnet_name(instance, subnetvnetname) {
 }
 
 fn azure_lb_frontend_ip_create(instance) {
-	(azure network lb frontend-ip create $instance)
+	azure network lb frontend-ip create $instance
```

Comment:
I think we can change to Az 2.0 here!

Reply:
I was on the ASAP mode =). Perhaps on a future refactoring.